README.md

```diff
@@ -138,12 +138,19 @@ The shared storage worklet invocation methods (`addModule`, `createWorklet`, and
     *   `key` and `value` are both strings.
     *   Options include:
         *   `ignoreIfPresent` (defaults to false): if true, a `key`’s entry is not updated if the `key` already exists. The embedder is not notified which occurred.
-*   `window.sharedStorage.append(key, value)`
+        * `withLockOnResource`: acquire a lock on the designated resource before executing. See [Locking for modifier methods](#locking-for-modifier-methods) for details.
```

Comment:
I'm partial to `withLock`. If we need a third word then I'd prefer, `withLockRequest` as that is closer to the lockmanager api.

Reply:
Done.